packages/dev/inspector-v2/src/components/properties/animation/animationGroupProperties.tsx

```diff
@@ -1,11 +1,97 @@
-import type { FunctionComponent } from "react";
+import type { AnimationGroup } from "core/Animations/animationGroup";
+import { useCallback, type FunctionComponent } from "react";
+import { ButtonLine } from "shared-ui-components/fluent/hoc/buttonLine";
+import { SyncedSliderPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/syncedSliderPropertyLine";
+import { TextPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/textPropertyLine";
+import { BoundProperty } from "../boundProperty";
+import { useObservableState } from "../../../hooks/observableHooks";
+import { SwitchPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/switchPropertyLine";
+import { Collapse } from "@fluentui/react-motion-components-preview";
+import { NumberInputPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/inputPropertyLine";
 
-import type { AnimationGroup } from "core/index";
+interface ICurrentFrameHolder {
+    currentFrame: number;
+}
 
 export const AnimationGroupControlProperties: FunctionComponent<{ animationGroup: AnimationGroup }> = (props) => {
-    return <></>;
+    const { animationGroup } = props;
+    const targetedAnimations = animationGroup.targetedAnimations;
+    let currentFrameHolder: ICurrentFrameHolder | undefined = undefined;
+
+    if (targetedAnimations.length > 0) {
+        currentFrameHolder = targetedAnimations[0].animation.runtimeAnimations.find((rA) => rA.target === targetedAnimations[0].target);
+    }
+
+    const currentFrame = useObservableState(
+        useCallback(() => {
+            return currentFrameHolder ? currentFrameHolder.currentFrame : undefined;
+        }, [currentFrameHolder]),
+        animationGroup.getScene().onBeforeRenderObservable
+    );
+
+    const isPlaying = useObservableState(
+        useCallback(() => {
+            return animationGroup.isPlaying;
+        }, [animationGroup]),
+        animationGroup.getScene().onBeforeRenderObservable

```

Comment:
Seems like you could use `onAnimationGroupPlayObservable` and `onAnimationGroupPauseObservable` (you can pass multiple observables to useObservableState).